admin-dev/themes/new-theme/js/pages/order/view/order-product-add.js

```diff
@@ -118,6 +119,7 @@ export default class OrderProductAdd {
       );
     });
     this.productAddActionBtn.on('click', event => this.addProduct($(event.currentTarget).data('orderId')));
+    this.productAddMenuBtn.on('click', () => this.orderProductRenderer.toggleProductAddNewInvoiceInfo());
```

Comment:
Is this necessary? Since it's already done in `OrderViewPage::listenForProductAdd`?
Isn't the fix in `toggleProductAddNewInvoiceInfo` enough?

Reply:
Yes you are right.